frontend/server/src/main/java/org/pytorch/serve/wlm/ModelManager.java

```diff
@@ -211,15 +214,34 @@ private void setupModelDependencies(Model model)
                             + requirementsFilePath; // NOPMD
 
             String[] envp =
-                    EnvironmentUtils.getEnvString(configManager.getModelServerHome(), null, null);
+                    EnvironmentUtils.getEnvString(model.getModelDir().getAbsolutePath(), null, null);
```

Comment:
I think getting the path from configManager was done purposefully so that a customer can provide a non-default python path. I am surprised there are no unit test failing for this change

Reply:
Actually No unit tests are failing due to change. I'm suggesting this change because of the following reasons: In other places where getEnvString() is called (e.g., [here](https://github.com/pytorch/serve/blob/4f7349a13e98a1f44f76db01af976ed00da5d468/frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerLifeCycle.java#L60) and [here](https://github.com/pytorch/serve/blob/f0da8d2714628cb5de0c6eaa3ac0448a6b226640/frontend/server/src/main/java/org/pytorch/serve/metrics/MetricCollector.java#L40) ) the 'cwd' argument is being set to the temporary closure of the model i.e. '/tmp/models/<random_id>'. So using 'configManager.getModelServerHome()' location seems like an anomaly in the setupModelDependencies function. 

Also, the value of 'configManager.getModelServerHome()' has the string 'site-packages' in it, which results in the value NOT being added to the PYTHONPATH as per this [line](https://github.com/pytorch/serve/blob/a95ef13eb735efba6168ed485d7ab943f5e65ac3/frontend/server/src/main/java/org/pytorch/serve/util/messages/EnvironmentUtils.java#L44). So the value seems to be wasted.

I don't see how using configManager.getModelServerHome() is allowing customers to provide a non-default path. It's value on an Ubuntu instance is: `configManager.getModelServerHome(): /home/ubuntu/test/lib/python3.6/site-packages`, whereas the value of model.getModelDir().getAbsolutePath() looks like: `model.getModelDir().getAbsolutePath(): /tmp/models/6615713b803846919386a82102e4fbed`.

This line change doesn't change anything significant as of now, and is introduced more for consistency and correctness, than for fixing a specific bug.

Reply:
"site-packages" is included in pip default value. That's why 'configManager.getModelServerHome()' has the string 'site-packages' in it. This value can be changed if "model_server_home" is set in property file. 

The sanity test didn't fall because [config_test_env.properties](https://github.com/pytorch/serve/blob/master/frontend/server/src/test/resources/#L6) and [config.properties](https://github.com/pytorch/serve/blob/master/frontend/server/src/test/resources/config.properties#L6) unset "model_server_home".

I think both configManager.getModelServerHome() and model.getModelDir().getAbsolutePath() should be added in pythonpath.

Reply:
Done